clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3855,48 +3855,6 @@ public DeleteConsumerGroupOffsetsResult deleteConsumerGroupOffsets(
         return new DeleteConsumerGroupOffsetsResult(future.get(CoordinatorKey.byGroupId(groupId)), partitions);
     }
 
-    private static final class ListShareGroupsResults {
-        private final List<Throwable> errors;
-        private final HashMap<String, ShareGroupListing> listings;
```

Comment:
Could you please remove `ShareGroupListing` as well?

Reply:
Done.